cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -19,20 +19,20 @@ package hostfolder
 import (
 	"fmt"
 	"runtime"
+	"strings"
 
-	"errors"
 	miniConfig "github.com/minishift/minishift/pkg/minishift/config"
 	hostFolderConfig "github.com/minishift/minishift/pkg/minishift/hostfolder"
 	"github.com/minishift/minishift/pkg/minishift/hostfolder/config"
 	"github.com/minishift/minishift/pkg/util"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
-	"github.com/minishift/minishift/pkg/util/strings"
+	miniStrings "github.com/minishift/minishift/pkg/util/strings"
```

Comment:
It should have been `minishiftStrings" like here https://github.com/minishift/minishift/search?utf8=%E2%9C%93&q=minishiftStrings&type= .

Reply:
@budhrg made it `miniString` because we have `miniConfig` in the same file here.. :)